applications/common/backend/src/main/resources/libs/kyanite/common/components/tabs/tabs.html

```diff
@@ -64,7 +64,34 @@
                     </sly>
                 </div>
             </div>
+            <ul class="tabs-mobile-container" data-sly-test="${!model.vertical}">
+                <sly data-sly-list="${resource.children}">
+                <li class="tab-mobile">
+                    <sly data-sly-test="${item.resourceType == 'kyanite/common/components/tabs/tab'}">
```

Comment:
Shouldn't this test be before `<li>` tag? If it has different type we will have empty `<li>`?

Reply:
I think it should. I didn't inspect the mobile layout, just moved it to the project as-is.